ctru-rs/src/services/gfx.rs

```diff
@@ -247,7 +296,7 @@ impl TopScreen {
 
 impl Screen for TopScreen {
     fn as_raw(&self) -> ctru_sys::gfxScreen_t {
-        self.left.as_raw()
+        ctru_sys::GFX_TOP
```

Comment:
oops, this was probably actually fine before, but I thought it was another mistake use of implicit conversion from `gfx3dSide_t` to `gfxScreen_t`.